exept/exeptions.py

```diff
@@ -83,4 +83,28 @@ class UserAlreadyExistError(CustomQSportException):
     def __init__(self, date_value):
         self.date_value = date_value
         message = f"User with such data already exist'"
+        super().__init__(message)
+
+
+class NoUsersInDBError(CustomQSportException):
```

Comment:
I would do just "UserNotFound"

This exception is basically impossible to get anyways. On real project you'd have AT LEAST some developer accounts in place or smth like that. Plust it'd expose we have no users to our API clients  :)